src/main/java/uk/ac/ebi/subs/ena/processor/ENAProcessor.java

```diff
@@ -169,5 +190,89 @@ private String centerName(SubmissionEnvelope submissionEnvelope) {
         return paramMap;
     }
 
+    private Study setDefaultStudyReleaseDate(SubmissionEnvelope submissionEnvelope, Study study) {
+        Date subDate = submissionEnvelope.getSubmission().getSubmissionDate();
+
+        if (subDate != null && !study.isAccessioned() && study.getReleaseDate() == null) {
+            study.setReleaseDate(convertSubmissionDate(submissionEnvelope.getSubmission().getSubmissionDate()));
```

Comment:
You could use `subDate` variable instead of getting it again from the envelop

```suggestion
            study.setReleaseDate(convertSubmissionDate(subDate));
```

Reply:
good catch. will fix this as well.

Reply:
just use the suggestion from github :-)